_data/sites.json

```
@@ -10310,6 +10310,17 @@
             "recargapay.workable.com"
         ]
     },
+    
+    {
+        "name": "Redacteur.com",
+        "url": "https://www.redacteur.com/identities/edit",
+        "difficulty": "easy",
+        "notes": "Login to your account. Click on \"My Account\" then \"My connexion information\". Finally click \"Delete my account\"",
```

```suggestion
        "notes": "Login to your account. Click on \"My Account\" then \"My connection information\". Finally click \"Delete my account\"",
```
Should it be connection?